snap7/client/__init__.py

```diff
@@ -1013,7 +1017,9 @@ def _prepare_as_read_area(self, area: Areas, size: int) -> Tuple[WordLen, Array]
         usrdata = (type_ * size)()
         return wordlen, usrdata
 
-    def as_read_area(self, area: Areas, dbnumber: int, start: int, size: int, wordlen: WordLen, pusrdata) -> int:
+    def as_read_area(
+        self, area: Areas, dbnumber: int, start: int, size: int, wordlen: WordLen, pusrdata: "Array[_SimpleCData[int]]"
```

Comment:
why does this type need to be a a string?

Reply:
Python interprets  `_SimpleCData[Any]` as indexing an element, which raises an exception . Therefore, it needs to be enclosed in quotes
![Screenshot from 2024-06-02 22-42-12](https://github.com/gijzelaerr/python-snap7/assets/52915342/eb065108-673f-4c12-aac8-13949d43f317)